pom.xml

```diff
@@ -4,94 +4,45 @@
   xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
   <artifactId>SalaryCalculator</artifactId>
 
+  <build>
+    <plugins>
+      <plugin>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <groupId>org.apache.maven.plugins</groupId>
+        <version>3.0.0-M7</version>
+      </plugin>
+    </plugins>
+  </build>
   <dependencies>
-    <dependency>
-      <artifactId>junit-jupiter-engine</artifactId>
-      <groupId>org.junit.jupiter</groupId>
-      <scope>test</scope>
-      <version>RELEASE</version>
-    </dependency>
-
     <dependency>
       <artifactId>junit-jupiter-params</artifactId>
       <groupId>org.junit.jupiter</groupId>
       <scope>test</scope>
       <version>5.8.2</version>
     </dependency>
-    <dependency>
-      <artifactId>testng</artifactId>
-      <groupId>org.testng</groupId>
-      <scope>test</scope>
-      <version>RELEASE</version>
-    </dependency>
-    <dependency>
-      <artifactId>junit</artifactId>
-      <groupId>junit</groupId>
-      <scope>test</scope>
-      <version>RELEASE</version>
-    </dependency>
     <dependency>
       <artifactId>assertj-core</artifactId>
       <groupId>org.assertj</groupId>
-      <!-- use 2.9.1 for Java 7 projects -->
       <scope>test</scope>
       <version>3.23.1</version>
     </dependency>
     <dependency>
       <artifactId>lombok</artifactId>
       <groupId>org.projectlombok</groupId>
       <scope>provided</scope>
-      <version>1.18.10</version>
+      <version>1.18.24</version>
     </dependency>
     <dependency>
       <artifactId>postgresql</artifactId>
       <groupId>org.postgresql</groupId>
-      <version>42.3.3</version>
-    </dependency>
-
-
-    <!-- https://mvnrepository.com/artifact/com.zaxxer/HikariCP -->
-    <dependency>
-      <artifactId>HikariCP</artifactId>
-      <groupId>com.zaxxer</groupId>
-      <version>5.0.1</version>
-    </dependency>
-    <dependency>
-      <artifactId>docx4j</artifactId>
-      <exclusions>
-        <exclusion>
-          <artifactId>slf4j-log4j12</artifactId>
-          <groupId>org.slf4j</groupId>
-        </exclusion>
-        <exclusion>
-          <artifactId>log4j</artifactId>
-          <groupId>log4j</groupId>
-        </exclusion>
-      </exclusions>
-      <groupId>org.docx4j</groupId>
-      <version>RELEASE</version>
-    </dependency>
-
-    <dependency>
-      <artifactId>slf4j-api</artifactId>
-      <groupId>org.slf4j</groupId>
-      <version>RELEASE</version>
+      <version>42.3.6</version>
     </dependency>
-    <dependency>
-      <artifactId>slf4j-simple</artifactId>
-      <groupId>org.slf4j</groupId>
-      <version>RELEASE</version>
-    </dependency>
-
-
   </dependencies>
   <groupId>org.example</groupId>
```

Comment:
groupId should be acoustic.com with some description, just a copy paste from production repo:
    <groupId>co.acoustic.commonsrv.user.management</groupId>
    <artifactId>commonsrv-user-management-service</artifactId>
    <version>0.0.16-SNAPSHOT</version>
    <name>commonsrv-user-management-service</name>
    <description>User management facade service</description>

Reply:
fixed